docs_src/general/ServicePorts.md

```diff
@@ -5,15 +5,19 @@ The following tables (organized by type of service) capture the default service
     |Services Name|	Port Definition|
     |---|---|
     |core-data|	59880|
-    |ZMQ - to be deprecated in a future release|5563|
     |core-metadata	|59881|
     |core-command	|59882|
+    |redis|6379|
+    |consul|8500|
+    |ZMQ - deprecated|5563|
```

Comment:
This is a Core Data port for ZMQ, hence it is under Core Data.

Reply:
Okay. It was already there. I'm just moving it down the list. Will update.